src/views/modules/meta/details.php

```diff
@@ -22,6 +22,31 @@
 $end_time = tribe_get_end_date( null, false, $time_format );
 $end_ts = tribe_get_end_date( null, false, Tribe__Date_Utils::DBDATEFORMAT );
 
+$formatted_time = null;
+if ( $start_time == $end_time ) {
+	$formatted_time = esc_html( $start_time );
+} else {
+	$formatted_time = esc_html( $start_time . $time_range_separator . $end_time );
+}
+
+$event_id = Tribe__Main::post_id_helper();
+
+/**
+ * Returns a formatted time for a single event
+ *
+ * @var string Formatted time string
+ * @var int Event post id
+ */
+$formatted_time = apply_filters( 'tribe_events_single_event_formatted_time', $formatted_time, $event_id );
```

Comment:
Why the filter name is swapped?`tribe_events_single_event_formatted_time` instead of `tribe_events_single_event_time_formatted`, I mean just to follow the filter below.